Documentation/platforms/arm64/bcm2711/boards/raspberrypi-4b/index.rst

```diff
@@ -139,17 +159,14 @@ Board Peripheral Support
 
 SMP is currently unsupported.
 
+To see support for general chip peripherals (I2C, SPI, UART, etc), see the
+:doc:`BCM2711 page <../../index>`
+
 NuttX for the Raspberry Pi 4 supports these on-board peripherals:
 
 ======================== =======
 Peripheral               Support
 ======================== =======
-I2C                      Partial (able to read, that's it)
-UART                     Mini UART yes, PL011 no
-GPIO                     Partial
-PWM                      No
-SPI                      No
-PCM                      No
 AV port                  No
 HDMI                     No
 WiFi                     No
```

Comment:
Maybe these parts should be removed since they were moved to another place

Reply:
I left the last few there since those features are specific to the board. Maybe HDMI could be moved but I'm actually not certain right now if the BCM2711 has an HDMI interface or if it interfaces with some other chip on the board